src/PAMI/Message/Event/Factory/Impl/EventFactoryImpl.php

```diff
@@ -69,7 +69,11 @@ public static function createFromRaw($message)
         for ($i = 0; $i < $totalParts; $i++) {
             $parts[$i] = ucfirst($parts[$i]);
         }
-        $name = implode('', $parts);
+        if (PHP_MAJOR_VERSION < 7 || (PHP_MAJOR_VERSION == 7 && PHP_MINOR_VERSION < 4)) {
```

Comment:
I think there is no need to check for version when the regular signature will work as well.